test/test_consistency.jl

```diff
@@ -0,0 +1,44 @@
+function check_quadratic_model(model, quadraticmodel)
+    rtol  = 1e-8
+    # @assert typeof(quadraticmodel) <: NLPModels.QuadraticModel
```

Comment:
Is this supposed to be commented out?

Reply:
Forgot that, changed to `typeof(quadraticmodel) <: QuadraticModels.AbstractQuadraticModel`